platform/ui/src/components/ViewportActionBar/ViewportActionBar.jsx

```diff
@@ -224,23 +219,36 @@ ViewportActionBar.propTypes = {
       scanner: PropTypes.string.isRequired,
     }),
   }).isRequired,
+  numCols: PropTypes.number,
 };
 
 ViewportActionBar.defaultProps = {
   showNavArrows: true,
   showPatientInfo: false,
+  numCols: 1,
 };
 
 function PatientInfo({
   patientName,
   patientSex,
   patientAge,
   MRN,
-  thickness,
+  thickness = 'N/A',
   spacing,
   scanner,
+  numCols,
   isOpen,
 }) {
+  const classes = {
+    infoHeader: 'text-base text-primary-light',
+    infoText: classnames(
+      'text-base text-white truncate',
+      numCols === 1 ? 'max-w-48' : 'max-w-16'
```

Comment:
I _really_ don't want this thing's width to be based on the reactive viewportGridService's number of columns state.
This should either be a fixed width regardless of the number of columns, or we should find a way for it to determine it's width relative to the viewportPane it's contained inside.